2024_Sasso_PNAS/2024_Sasso_PNAS.janno

```diff
@@ -0,0 +1,49 @@
+Poseidon_ID	Genetic_Sex	Group_Name	Relation_To	Relation_Degree	Relation_Type	Collection_ID	Country	Country_ISO	Location	Site	Latitude	Longitude	Date_Type	Date_C14_Labnr	Date_C14_Uncal_BP	Date_C14_Uncal_BP_Err	Date_BC_AD_Start	Date_BC_AD_Median	Date_BC_AD_Stop	Date_Note	MT_Haplogroup	Y_Haplogroup	Source_Tissue	Nr_Libraries	Library_Names	Capture_Type	UDG	Library_Built	Genotype_Ploidy	Data_Preparation_Pipeline_URL	Endogenous	Nr_SNPs Ps	Damage	Contamination	Contamination_Err	Contamination_Meas	Contamination_Note	Genetic_Source_Accession_IDs	Publication	Eager_ID	Main_ID	RateErrX	RateErrY	RateX	RateY
```

Comment:
The `Nr_SNPs Ps` column should be called `Nr_SNPs`. I hope this doesn't affect other packages as well?

Reply:
Good catch! Checked, and this is only in this package. Must have introduced this while manually editing.